### PR TITLE
returns login status

### PR DESCRIPTION
**Pull request by davidvargaspuga**
returns the login status of the user.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    String userEmail = userService.getCurrentUser().getEmail();
+    String logoutUrl = userService.createLogoutURL("/");
+    out.println("<h1>Home</h1>");
+    out.println("<p>Hello " + userEmail + "!</p>");
+    out.println("<p>Logout <a href=\"" + logoutUrl + "\">here</a>.</p>");
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
```
Comment (marcelpuyat) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:58]: It'll be good to get in the habit of leaving comments for things not yet implemented, i.e. something like:
```
// TODO(David): Implement _.
```
^In this case, fill in the functionality that will occur when a POST request is made to /login.

This way, collaborators will have an easier time working with your checked in code.

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    if(!userService.isUserLoggedIn()){
+        String loginUrl = userService.createLoginURL("/login");
+        out.println("<p>Login <a href=\"" + loginUrl + "\">here</a>.</p>");
+        return;
+    }
+
+    // User is logged in 
+    String userEmail = userService.getCurrentUser().getEmail();
+    String logoutUrl = userService.createLogoutURL("/");
+    out.println("<h1>Home</h1>");
```
Comment (marcelpuyat) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:51]: I assume you'll eventually send the user to an html page? Or is this the final solution here (to print out html strings in the Java file)?

**Review by marcelpuyat — CHANGES_REQUESTED** [review]
Good job getting an early start to week 3, David! I just have one question to address and one suggestion for you.

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 2 above)
Comment (davidvargaspuga) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:51]: yes, ill redirect to a page that is displayed before a user is logged in

Comment (marcelpuyat) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:51]: Sounds good.

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 1 above)
Comment (marcelpuyat) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:58]: Once this is resolved, feel free to merge this PR.

**Pull request merged**